gym/lib/assets/wrap_xcodebuild/xcbuild-safe.sh

```diff
@@ -41,6 +44,22 @@ if [[ $? -eq 0 ]]; then
   unset BUNDLE_GEMFILE
 fi
 
+if command_exists rbenv; then
+  # Cause rbenv to use system ruby. Lasts only for the scope of this
+  # session which will normally just be this script.
+  rbenv shell system
+
+  # Unset environment variables that could break xcodebuild
+  unset RUBYLIB
+  unset RUBYOPT
+  uset _ORIGINAL_GEM_PATH
```

Comment:
Oooops
On May 2, 2016 18:39, "Oleksii Taran" <notifications@github.com> wrote:

> In gym/lib/assets/wrap_xcodebuild/xcbuild-safe.sh
> <https://github.com/fastlane/fastlane/pull/4456#discussion_r61764092>:
>
> > @@ -41,6 +44,22 @@ if [[ $? -eq 0 ]]; then
> >    unset BUNDLE_GEMFILE
> >  fi
> >
> > +if command_exists rbenv; then
> > +  # Cause rbenv to use system ruby. Lasts only for the scope of this
> > +  # session which will normally just be this script.
> > +  rbenv shell system
> > +
> > +  # Unset environment variables that could break xcodebuild
> > +  unset RUBYLIB
> > +  unset RUBYOPT
> > +  uset _ORIGINAL_GEM_PATH
>
> Typo? uset vs unset
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/fastlane/fastlane/pull/4456/files/5617dfb393ca5f5f56bd5d277b26a8c84db20149#r61764092>
>